vignettes/trap_effort.Rmd

```diff
@@ -105,22 +113,22 @@ in nonsensical hours fished. In these cases, we assume 24 hour sampling periods.
 
 ## Hours Fished {.tabset}
 
-## Method 1: Battle Creek, Clear Creek, Feather River, Knights Landing, Tisdale
-
-The above creeks have data available to calculate the hours fished. There are a few
-cases when start date is NA. For these we fill in the start date assuming
-it is the sample date of the previous record. Note that there are some records with 0 hours.
+We now process all locations together using a combination of method 1 and 3. 
+All locations currently have both a start and stop date though this was not the case
+historically. There are cases when start date or stop date is NA. 
+For these we fill in the start date assuming it is the sample date of the previous record. When
+the stop date is missing we assume it is the start date of the next record.
+Note that there are some records with 0 hours.
 These occur because trap visit data was entered when the trap was not in service and
-then a new trap visit entry was recorded once the trap started working again.
+then a new trap visit entry was recorded once the trap started working again. There
+are some erroneous values. These will be addressed at the end of the script but setting limits.
```

Comment:
*at the end of the script **by** setting limits I think

Reply:
done